projectPage.css

```diff
@@ -1,57 +1,36 @@
-.vid {
+@import url('https://fonts.googleapis.com/css2?family=Abril+Fatface&family=Oswald:wght@500&display=swap');
+
+
+/* HEADER 
+=================================================*/
+.header-vid {
     object-fit: cover;
-    width: 100vw;
+    width: 100%;
     height: 200px;
     position: fixed;
-   
   }
 
-  /* @media screen and (max-width: 600px) {
-    #vid {
-      width: 36%;
-      height: 7%;
-    }
-  } */
-
 
-h1 {
+.header-h1 {
+    /* Text positioning */
     position: fixed;
-    flex-direction: row;
+    margin-top: auto;
+    width: 100%;
+    padding: 0;
+
+    /* Text formatting */
     text-shadow: 6px 6px 6px   rgb(53, 50, 50) ;
     color: #fff;
     font-family: 'Oswald', sans-serif;
-    /* mix-blend-mode:overlay; */
-    /* mix-blend-mode: multiply; */
-    margin-top: auto;
-    margin-bottom: auto;
-    font-size: 115px;
-    margin-left: auto;
-    margin-right: auto;
-    height: 20%;
-    width: 100%;
+    font-size: 7rem;
+    line-height: 200px;
     text-align: center;
-    padding: 0;
-}
 
-.pro {
+    /* Other text effects*/
     mix-blend-mode:overlay;
-    height: 10%;
-    position: fixed;
-
+    text-transform: uppercase;
```

Comment:
rather than making text ALL CAPS in your html itself, use text-transform uppercase for stylistic/decorative capital letters. using ALL CAPS in the html itself can cause screen readers to read it differently.